topic_classifier/test_from_file.py

```
@@ -0,0 +1,56 @@
+    """
+    data_dir = os.path.join("data")
+    data_path = os.path.join(data_dir, DATA_NAME)
+    if not os.path.exists(data_dir):
+        raise FileNotFoundError(f"Дирректория '{data_dir}' не найдена.")
+    if not os.path.isfile(data_path):
+        raise FileNotFoundError(f"Файл '{data_path}' не найден.")
+
+    try:
+        df = pd.read_csv(data_path)
+    except Exception as e:
+        raise ValueError(f"Произошла ошибка при чтении файла: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)